tests/lib/sms/run_coverage

```diff
@@ -0,0 +1,10 @@
+# This will compile and instrument code for gcov code coverage run.
```

Comment:
This is something new, @carlescufi @tejlmand do we accept such helper scripts in the repository?

Reply:
Just let me know what to do and I'll remove it if needed.

Reply:
isn't this already covered by twister ?
```
$ ./scripts/twister -h
...
 --enable-coverage     Enable code coverage using gcov.
  -C, --coverage        Generate coverage reports. Implies --enable-coverage.
  --coverage-platform COVERAGE_PLATFORM
                        Plarforms to run coverage reports on. This option may
                        be used multiple times. Default to what was selected
                        with --platform.
  --gcov-tool GCOV_TOOL
                        Path to the gcov tool to use for code coverage reports
  --coverage-tool {lcov,gcovr}
                        Tool to use to generate coverage report.
  --coverage-basedir COVERAGE_BASEDIR
                        Base source directory for coverage report.
```

and to my knowledge, ``lcov` is the default.

So I don't believe this is needed.

Reply:
I've removed this file